content/docs/quickstart/sqlalchemy.md

```diff
@@ -21,47 +31,54 @@ CONNSTR = f'postgresql://{USERNAME}:{TOKEN}@{PROJECT_ID}.cloud.neon.tech/main'
 engine = create_engine(CONNSTR)
 ```
 
-References:
+## References
 
 - [Establishing Connectivity - the Engine](https://docs.sqlalchemy.org/en/14/tutorial/engine.html)
 - [Connecting to PostgreSQL with SQLAlchemy](https://docs.sqlalchemy.org/en/14/core/engines.html#postgresql)
 
-### Using from python + psycopg2
-
-Psycopg2 is the most popular python library for running raw postgres queries. If you’re interested in a higher-level ORM on top of psycopg2, see our guides on [SQLAlchemy](#using-from-sqlalchemy) and [Django](#using-from-django).
-
-To get started writing postgres queries against neon via psycopg2:
-
-1. Register on Neon cloud service and create a project
-2. Navigate to your Project on console.neon.tech and find the Postgres Username and access token in the “Connection Details” section.
-3. Install psycopg2. You might also need psycopg2-binary depending on your system. You can run “pip install psycopg2 psycopg2-binary” or use a dependency manager like poetry to do the same.
-4. Run the “hello neon” program:
-    
-    ```python
-    import psycopg2
-    
-    # Optional: tell psycopg2 to cancel the query on Ctrl-C
-    import psycopg2.extras; psycopg2.extensions.set_wait_callback(psycopg2.extras.wait_select)
-    
-    # NOTE: the password can be set to None if it's specified in the ~/.pgpass file
-    USERNAME = "<your-username>"
-    ACCESS_TOKEN = "<your-access-token>"
-    HOST = "pg.neon.tech"
-    PORT = "5432"
-    PROJECT = "main"
-    
-    conn = psycopg2.connect(
-     host=HOST,
-     port=PORT,
-     user=USERNAME,
-     password=ACCESS_TOKEN,
-     database=PROJECT)
-    
-    with conn.cursor() as cur:
-     cur.execute("SELECT 'hello neon';")
-     print(cur.fetchall())
-    ```
-    
-5. Build great things with Neon! Any postgres tutorial will be able to guide you on the syntax.
-
-Note: This example was tested with python 3 and psycopg2 version 2.9.3
+## Using from Python + psycopg2
+
+Psycopg2 is the most popular python library for running raw postgres queries. If you are interested in a higher-level ORM on top of psycopg2, see our guides on [SQLAlchemy](#using-from-sqlalchemy) and [Django](#using-from-django).
+
+This step will cover how to get started with writing postgres queries against Neon via psycopg2.
+
+First, register on the Neon cloud service and create a Project.
+
+Next, navigate to your Project on [console.neon.tech](console.neon.tech/).
+
+Then, install psycopg2. You may also need psycopg2-binary depending on your system. To install psycopg2-binary, You can run “pip install psycopg2 psycopg2-binary” or use a dependency manager like [Poetry](https://python-poetry.org/) to do the same.
```

Comment:
I think it's better to embed commands like `pip install` in backticks so they can be easily copied. But this should probably be covered somewhere in the style guide.

Reply:
Changed in following commit : https://github.com/neondatabase/website/pull/59/commits/13211184f92957eee8b6b4e8680b330781e6dac5